helix-term/src/ui/editor.rs

```diff
@@ -444,11 +444,17 @@ impl EditorView {
             // extra loops if the code is deeply nested.
 
             for i in starting_indent..(indent_level / tab_width as u16) {
+                let style = if config.indent_guides.rainbow {
+                    indent_guide_style.patch(theme.get_rainbow(i as usize))
```

Comment:
This may look odd if `rainbow = true` is set as well as `skip-levels = n` for some non-zero n. Currently if you skip one level, the first level uses the second color of the rainbow but I think we should start the rainbow on the first indent guide

Reply:
At first glance this looks like it would be weird, but this behaviour causes an interesting side-effect together with your rainbow brackets.
It matches the color based on indentation.
consider this:
![image](https://user-images.githubusercontent.com/13474089/195799819-06b0644e-de04-4074-b983-4f4dad8263b8.png)

if i set the linenumber skip for the indent guides now to let's say 2, it would still line up
![image](https://user-images.githubusercontent.com/13474089/195799969-32a05d2d-e22d-4963-a886-cd75c1c8c4e6.png)

with the fix, this alignment would be lost.